src/phoenix/trace/llama_index/callback.py

```diff
@@ -18,21 +17,45 @@
 
 
 def _check_instrumentation_compatibility() -> bool:
-    if find_spec("llama_index") is None:
-        raise PackageNotFoundError("Missing `llama-index`. Install with `pip install llama-index`.")
-    # split the version string into a tuple of integers
-    llama_index_version_str = version("llama-index")
-    llama_index_version = tuple(map(int, llama_index_version_str.split(".")[:3]))
+    llama_index_version_str = None
+    try:
+        llama_index_version_str = version("llama-index")
+    except PackageNotFoundError:
+        pass
+
+    llama_index_core_version_str = None
+    try:
+        llama_index_core_version_str = version("llama-index-core")
+    except PackageNotFoundError:
+        pass
```

Comment:
```suggestion
    llama_index_version_str = None
    llama_index_core_version_str = None
    try:
        llama_index_version_str = version("llama-index")
    except PackageNotFoundError:
        pass

    try:
        llama_index_core_version_str = version("llama-index-core")
    except PackageNotFoundError:
        pass
```

alternatively you could just make a function `get_packege_version(package_name: str): Optional[str]` that does this. I think that would read cleaner.